modules/renter/repair.go

```diff
@@ -30,6 +30,19 @@ var renewThreshold = func() types.BlockHeight {
 	}
 }()
 
+// uptimeWindow specifies the duration in which host uptime is checked.
+var uptimeWindow = func() time.Duration {
+	switch build.Release {
+	case "dev":
+		return 24 * time.Hour // 1 day
+	case "standard":
+		return 7 * 24 * time.Hour // 1 week
+	case "testing":
+		return 1 * time.Minute // 1 minute
+	}
```

Comment:
same changes apply to these constants as apply to the contractor constants - day is too long for "dev", etc.